microk8s-resources/actions/disable.ingress.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Ingress"
+
+ARCH=$(arch)
+TAG="0.15.0"
+if [ "${ARCH}" = arm64 ]
+then
+  TAG="0.11.0"
+fi
+cat "${SNAP}/actions/ingress.yaml" | \
+"$SNAP/bin/sed" 's@\$ARCH@'"$ARCH"'@g' | \
+"$SNAP/bin/sed" 's@\$TAG@'"$TAG"'@g' | \
```

Comment:
Would be nice for `use_manifest` to support this, maybe as an option, just for consistency.  Not sure how to do it in a generic way, though.  Maybe a largish `get_version_tag` helper or something?